packages/esm-service-queues-app/src/hooks/useSystemSetting.ts

```diff
@@ -0,0 +1,23 @@
+import { openmrsFetch, restBaseUrl } from '@openmrs/esm-framework';
+import useSWRImmutable from 'swr/immutable';
+
+export interface SystemSetting {
+  uuid: string;
+  property: string;
+  value: string;
+}
+
+export function useSystemSetting(setting: string) {
+  const apiUrl = `${restBaseUrl}/systemsetting?&v=custom:(value)&q=${setting}`;
```

Comment:
It may be less ambiguous to use:

```suggestion
  const apiUrl = `${restBaseUrl}/systemsetting/${setting}?v=custom:(value)`;
```

Since `q` is a prefix search.

Reply:
Fixed, though I couldn't commit this as-is without changing the handling from a List to single value.